LondonBritishLibrary/orient/BLorient827.xml

```diff
@@ -423,76 +479,76 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                     </dimensions>
                                 </extent>
-                                
-                                
+
+
                             </supportDesc>
-                            
+
                             <layoutDesc>
                                 <layout columns="2" writtenLines="23 25" cert="high"/>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
                                 <desc>The manuscript is neatly written of the <date>18th century.</date>
                                 </desc>
-                                
+
                                 <date notBefore="1700" notAfter="1800" resp="PRS10171Wright"/>
-                                
+
                             </handNote>
-                            
-                        </handDesc>   
+
+                        </handDesc>
                         <additions>
-                            <list> 
+                            <list>
                         <item xml:id="a1">
                                 <locus target="#4r"/>
                                 <desc type="OwnershipNote">the name of the  owner of the manuscript has been erased</desc>
                                 <q xml:lang="gez"/>
                             </item>
                             </list>
-                            <list> 
+                            <list>
                         <item xml:id="a2">
                                 <locus target="#40r"/>
                                 <desc type="OwnershipNote">the name of the  owner of the manuscript has been erased</desc>
                                 <q xml:lang="gez"/>
                             </item>
                                 <item xml:id="a3">
                                     <locus target="#4r"/>
-                                    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of 
+                                    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
                                         <placeName ref="INS0101MadhaneAlam"/>.</desc>
                                     <q xml:lang="gez">ዜና፡ እስክንድር፡ ዘ<placeName ref="INS0101MadhaneAlam">ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName>
                            </q>
                                 </item>
-                           
-                                  
+
+
                                        <item xml:id="e1">
                                            <locus from="1r" to="3r"/>
                                            <desc>trials of the pens and a few notes in Amharic.</desc>
                                            <q xml:lang="gez">ባርካ፡ እግዚኦ፡ ለዛቲ፡ ብርዕ፡
```

Comment:
Trials

Reply:
Sorry, I do not understand. I should delete "Pen" at the beginning?